src/dishes.js

```diff
@@ -0,0 +1,86 @@
+import firstImge from './assets/pexels-dhyamis-kleber-3584317.jpg';
+import secondImage from './assets/pexels-fadime-erbass-11572734.jpg';
+import thirdImage from './assets/pexels-chevanon-photography-312418.jpg';
+import forthImage from './assets/pexels-brett-jordan-825661.jpg';
+import fifthImage from './assets/pexels-tookapic-7782.jpg';
+import sixImage from './assets/pexels-tookapic-9708.jpg';
+
+
+export function dishes() {
+    const contentDiv = document.querySelector('#content');
+
+    contentDiv.replaceChildren();
+
+    const dishesContainer = document.createElement("div");
+    dishesContainer.classList.add("dishescont");
+
+    const firstDish = document.createElement("div");
+    firstDish.classList.add("firstDish");
+    const firstImg = document.createElement("img");
+    firstImg.classList.add("first-dish-imag");
+    firstImg.src = firstImge;
+    firstDish.textContent = "Black Tea as low as \n $2";
```

Comment:
- [ ] It'll be good if the picture sizes are compressed to allow them to load faster

Reply:
- [ ] The pictures are not currently aligned, kindly adjust them
![Screenshot (384)](https://user-images.githubusercontent.com/86371332/174452903-1f989afe-0a17-4e7b-9f2a-7045599468d8.png)

![Screenshot (385)](https://user-images.githubusercontent.com/86371332/174452885-059e14d6-681b-49ae-945d-0c41c7967d90.png)

Reply:
this how it displaying at my end
![Screenshot (27)](https://user-images.githubusercontent.com/65237847/174615317-441e375c-ca9a-4e2f-98a5-f2c8b3ff2568.png)

![Screenshot (28)](https://user-images.githubusercontent.com/65237847/174615011-e063682b-12f9-432c-a339-147db8f3b525.png)

Reply:
Noted